_includes/1.0/api/creditmemo.html

```diff
@@ -0,0 +1,205 @@
+                <td>The billing address. The Contact address if any, the Customer one otherwise.</td>
+            </tr>
+            <tr>
+                <td>tax_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>US only.<br />Id of the <a href="/docs/1.0/api/#tax" title="">Tax</a> to apply on that invoice.</td>
+            </tr>
+            <tr>
+                <td>reduce_rate</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>The reduce rate (%) to apply. From 0 to 100.<br />Default: 0.00</td>
+            </tr>
+            <tr>
+                <td>order_lines</td>
+                <td>{{ site.format.type.order_line }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>An array containing the order lines of the invoice.</td>
+            </tr>
+            <tr>
+                <td>public_note</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>Comment that will be printed on the final document.</td>
+            </tr>
+            <tr>
+                <td>shipping_contact_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>Id of the <a href="/docs/1.0/api/#contact" title="">Contact</a> the goods should be shipped to, if different than the Customer.<br />Could be any Contact from any Customer.</td>
+            </tr>
+            <tr>
+                <td>shipping_address</td>
+                <td>{{ site.format.type.address }}</td>
+                <td>no</td>
+                <td>yes</td>
+                <td>The shipping address. The shipping Contact address if any, the Customer one otherwise.</td>
+            </tr>
+            <tr>
+                <td>payment_terms_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>Id of the <a href="/docs/1.0/api/#paymentterm" title="">Payment Term</a> to apply on that invoice. Will be used as a template to generate Settlements.</td>
+            </tr>
+            <tr>
+                <td>project_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>Id of the <a href="/docs/1.0/api/#project" title="">Project</a> related to that invoice, if any.</td>
+            </tr>
+            <tr>
+                <td>user_code</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>no</td>
+                <td>no</td>
+                <td>The number assigned to that invoice (i.e. INV-002). If you don't specify an order number, it will be automatically generated.<br />64 characters max.</td>
```

Comment:
Credit Memo. Also update the user code accordingly (It's not INV-002 for Credit Memos)